src/devices/niko.ts

```diff
@@ -23,14 +56,21 @@ const local = {
             },
         } satisfies Fz.Converter,
         switch_action: {
-            cluster: 'manuSpecificNiko2',
+            cluster: 'manuSpecificNikoState',
             type: ['attributeReport', 'readResponse'],
             convert: (model, msg, publish, options, meta) => {
                 const state: KeyValue = {};
 
+                if (msg.data.switchActionReporting !== undefined) {
+                    const actionReportingMap: KeyValue = {0x00: false, 0x1f: true};
+                    state['action_reporting'] = utils.getFromLookup(msg.data.switchActionReporting, actionReportingMap);
+                }
                 if (msg.data.switchAction !== undefined) {
                     // NOTE: a single press = two separate values reported, 16 followed by 64
                     //       a hold/release cycle = three separate values, 16, 32, and 48
+                    // NOTE: these values should be interpreted bitwise
+                    //       when pushing multiple buttons at the same time, multiple bits can be set simultaneously and should generate multiple events
+                    //       currently, these values are not mapped and thus ignored
```

Comment:
@sjorge @Koenkk Is this possible to implement in the converter? It seems 1 value always maps to 1 'event', but in this case it could map to multiple 'events' when pushing multiple buttons at once.

Reply:
My first reaction would be to have the actions end up in a list but I don't think that is an excepted value for action property.

The next best thing would be to decode them into a list, then if there are > 1 in there merge them and add these as extra possible values.

Reply:
Would publishing multiple actions make sense here? To do that call `publish({"action": "x"})` here.

Reply:
Thanks, implemented it this way!